atd-vze/src/containers/DefaultLayout/DefaultHeader.js

```diff
@@ -19,64 +20,66 @@ const propTypes = {
-            src: logo,
-            width: 140,
-            height: 25,
-            alt: "Vision Zero Austin",
-          }}
-          // minimized={{ src: sygnet, width: 30, height: 30, alt: "CoreUI Logo" }}
-        />
-        <AppSidebarToggler className="d-md-down-none" display="lg" />
+  return (
+    <React.Fragment>
+      <AppSidebarToggler className="d-lg-none" display="md" mobile />
+      <AppNavbarBrand
+        full={{
+          src: logo,
+          width: 140,
+          height: 25,
+          alt: "Vision Zero Austin",
+        }}
+      />
+      <AppSidebarToggler className="d-md-down-none" display="lg" />
 
-        <Nav className="d-md-down-none" navbar>
-          <NavItem className="px-3">
-            <NavLink to="/dashboard" className="nav-link">
-              Dashboard
-            </NavLink>
-          </NavItem>
-          <NavItem className="px-3">
-            <NavLink to="/users" className="nav-link">
-              Users
-            </NavLink>
-          </NavItem>
-        </Nav>
-        <Nav className="ml-auto" navbar>
-          <UncontrolledDropdown nav direction="down">
-            <DropdownToggle nav>
-              <img
-                src={"./assets/img/avatars/1.png"}
-                className="img-avatar"
-                alt="admin@bootstrapmaster.com"
-              />
-            </DropdownToggle>
-            <DropdownMenu right>
-              <DropdownItem header tag="div" className="text-center">
-                <strong>Account</strong>
-              </DropdownItem>
-              <DropdownItem href="#/profile">
+      <Nav className="d-md-down-none" navbar>
+        <NavItem className="px-3">
+          <NavLink to="/dashboard" className="nav-link">
+            Dashboard
+          </NavLink>
+        </NavItem>
+        <NavItem className="px-3">
+          <NavLink to="/users" className="nav-link">
+            Users
+          </NavLink>
+        </NavItem>
+      </Nav>
+      <Nav className="ml-auto" navbar>
+        <UncontrolledDropdown nav direction="down">
+          <DropdownToggle nav>
+            <img
+              src={"./assets/img/avatars/1.png"}
+              className="img-avatar"
+              alt="admin@bootstrapmaster.com"
+            />
+          </DropdownToggle>
+          <DropdownMenu right>
+            <DropdownItem header tag="div" className="text-center">
+              <strong>Account</strong>
+            </DropdownItem>
+            <DropdownItem>
+              <Link
+                to="/profile"
+                style={{ color: "inherit", textDecoration: "inherit" }}
+              >
                 <i className="fa fa-user" /> Profile
-              </DropdownItem>
-              <DropdownItem onClick={logout}>
-                <i className="fa fa-lock" /> Log Out
-              </DropdownItem>
-            </DropdownMenu>
-          </UncontrolledDropdown>
-        </Nav>
-      </React.Fragment>
-    );
-  }
-}
+              </Link>
```

Comment:
For some reason, the profile button stopped working for me in Chrome, though it does work in Firefox. If it works for you in chrome, ignore this comment.

Reply:
Awesome catch - I had the same issue. Looks like it was a problem with the `react-router` `Link` component being nested in this dropdown. I changed it back to the original code, and it is working reliably again.